package/gcnano-binaries/Config.in

```diff
@@ -27,6 +27,26 @@ config BR2_PACKAGE_GCNANO_BINARIES
 
 if BR2_PACKAGE_GCNANO_BINARIES
 
+choice
+	prompt "platform"
+	default BR2_PACKAGE_GCNANO_BINARIES_PLATFORM_STM32MP1
+
+	comment "stm32mp1 support requires an arm 32-bits target"
+		depends on !BR2_arm
+
+	config BR2_PACKAGE_GCNANO_BINARIES_PLATFORM_STM32MP1
+	bool "stm32mp1"
+	depends on BR2_arm
+
+	comment "stm32mp2 support requires an arm64 target"
+		depends on !BR2_aarch64
+
+	config BR2_PACKAGE_GCNANO_BINARIES_PLATFORM_STM32MP2
+	bool "stm32mp2 (OEM release)"
+	depends on BR2_aarch64
+
+endchoice
+
```

Comment:
Just wondering if it is useful to add two separate configs as both can't ever be selected at the same time. We could just check the architecture in the gcnano-binaries.mk file. @tpetazzoni what do you think?
Otherwise, it seems good to me.

Reply:
I've implemented this change in the feature branch.

Reply:
Can you squash the 2 commits into one?